### PR TITLE
WINC-551: Updated the autoscaling documentation

### PR DESCRIPTION
**Pull request by EricPonvelle**
This ticket addresses the changes in [WINC-551](https://issues.redhat.com/browse/WINC-551) - https://github.com/openshift/windows-machine-config-operator/pull/279

With these changes, I created a new model that specifies how the machineset needs to be configured for autoscaling.

Versions: 4.7+

Preview: https://deploy-preview-32773--osdocs.netlify.app/openshift-enterprise/latest/machine_management/applying-autoscaling.html#cluster-autoscaler-machinesetapplying-autoscaling

### CODE REVIEW

File: modules/cluster-autoscaler-machineset.adoc [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+* `node-role.kubernetes.io/worker:``
++
+[IMPORTANT]
+====
+If these labels are not added, the the Windows node will not be marked as a `worker` node.
+====
+
+.Procedure
+
+. Configure the machineset template with values from your environment:
```
Comment (EricPonvelle) [inline modules/cluster-autoscaler-machineset.adoc:26]: This example is the vSphere template. I would like to combine Azure, vSphere, and AWS into one example if that is possible.

File: modules/cluster-autoscaler-machineset.adoc [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+          workspace:
+             datacenter: <vCenter DataCenter Name> <4>
+             datastore: <vCenter Datastore Name> <5>
+             folder: <Path to VM Folder in vCenter> # e.g. /DC/vm/ocp45-2tdrm <6>
+             server: <vCenter Server FQDN/IP> <7>
+----
+
+<1> The <infrastructureID> tag should be replaced with the output of the following command:
++
+`$ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster`
```
Comment (EricPonvelle) [inline modules/cluster-autoscaler-machineset.adoc:83]: I would appreciate someone providing an example output so that I can include that our users.

**Comment by EricPonvelle** [comment]
@aravindhp @sebsoto PTAL at these changes. For this ticket, I simply linked users to the existing machine set documentation during the configuration of the autoscaling. Please let me know if this covers the needs of WINC-551, and I'll get it reviewed and merged.

Also, I am unsure who to tag for QE review.

File: machine_management/applying-autoscaling.adoc [hunk 3]
```diff
@@ -32,9 +32,43 @@ include::modules/cluster-autoscaler-cr.adoc[leveloffset=+2]
 :FeatureResourceName: ClusterAutoscaler
 include::modules/deploying-resource.adoc[leveloffset=+2]
 
-== Next steps
+//
+// Addresses WINC-551 - https://github.com/openshift/openshift-docs/pull/32773 - Delete before merge
+//
 
-* After you configure the cluster autoscaler, you must configure at least one machine autoscaler.
+[id="cluster-autoscaler-machineset"]
```
Comment (aravindhp) [inline machine_management/applying-autoscaling.adoc:39]: Not sure if this level of detail is required. Just calling out that this also works for Windows nodes should be sufficient. We can also mention this in the 2.0.1 release notes.

**Comment by aravindhp** [comment]
/cc @sgaoshang @rrasouli 
for QE review

**Pull request closed without merging**